austation/code/game/decastation.dm

```diff
@@ -0,0 +1,54 @@
+/datum/map_template/shuttle/arrival/deca
+	suffix = "deca"
+	name = "arrival shuttle (Deca)"
+
+/obj/item/paper/fluff/stations/deca/bar
+	name = "Bar Memo"
+	info = "Yeah, I know the bar isn't done yet, I'm working on it! None of you touch those holofans or I'm putting your head on a stake.<br/>\
+	<br/>\
+	- Tania"
+
+/obj/item/paper/fluff/stations/deca/engine
+	name = "Engine Memo"
+	info = "In an effort to reduce spontanous station destruction at the hands of incompetent or non-existent engineers, you get this big ass solar panel instead.\
+	 Order an engine from cargo if you want and build it in atmos, but you get this solar panel for now.<br/>\
+	<br/>\
+	- Tania"
+
+/obj/item/paper/fluff/stations/deca/production
+	name = "Production Memo"
+	info = "Cutting costs and space where possible, the protolathe is now a singular unit which shares it's designs from all departments.\
+	 You can print whatever you want here, but the CE or RD can shut the shutters if they believe the protolathe is being abused.\
+	 Stay productive.<br/>\
+	<br/>\
+	- Tania"
+
+/obj/item/paper/fluff/stations/deca/rnd
+	name = "RnD Memo"
+	info = "Looking for the RnD console? It's in the experimentation lab. Or just hack this console and set it to allow research. Either way.<br/>\
+	<br/>\
+	- Tania"
+
+/obj/item/paper/fluff/stations/deca/tcomms
+	name = "Tcomms Memo"
+	info = "Ick keeps yelling at me that the station needs a minisat before it can operate. I'm working my ass off already, give me a break!\
+	 I'll have tcomms up eventually, for now this temporary setup should work fine.<br/>\
+	<br/>\
+	- Tania"
+
+/obj/item/paper/fluff/stations/deca/turbine
+	name = "Turbine Memo"
+	info = "Since you guys and atmos both do burns for tritium, I've decided to make it so you now share the turbine.\
+	 Go use it to make your tritium, or don't, but it ain't my problem.<br/>\
+	<br/>\
+	- Tania"
+
+/obj/structure/fluff/deca/tcomm
+	name = "Tania's Solution"
+	desc = "A rather messy assembly of various telecommunication components to substitute a dedicated telecommunications mainframe. By the looks of things, it's finally given up."
+	icon = 'icons/obj/machines/telecomms.dmi'
+	icon_state = "comm_server_o_off"
```

Comment:
@TheFakeElon 
what's your opinion on mapped character references?
I'm against it personally

Reply:
>no doris mention

remove it